src/FixedDataTableBufferedRows.js

```diff
@@ -101,7 +101,7 @@ var FixedDataTableBufferedRows = createReactClass({
   },
 
   _updateBuffer() {
-    if (this.isMounted()) {
+    if (this._rowBuffer) {
```

Comment:
I don't know if this is a safe fix yet. The usage of isMounted scattered through seems scary